### DISCUSSION TITLE
BambuHMI V1.0 - Centralized Remote X1C running on X1Plus Firmware and HomeAssistant.

### DISCUSSION THREAD

**Discussion by christophergault198** [X1Plus/X1Plus#185] [Show and tell]
Hello, wanted to briefly share a suite of tools and services that streamline my experience with the X1C that is now running X1Plus firmware. Attached are a couple screenshots, currently working on refactoring the codebase for a single config file that will allow a simple list of global variables to more easily connect to an independent printer and home assistant server (not just my printer, haha).

I will share files based on interest. Please give feedback! Thanks!
![alltools](https://github.com/X1Plus/X1Plus/assets/116657484/947a63f6-c48d-4a06-b3cc-f83c6397383b)
![coretools](https://github.com/X1Plus/X1Plus/assets/116657484/387aedf3-b668-4a0a-a1eb-b4d2bf929238)
![flow](https://github.com/X1Plus/X1Plus/assets/116657484/bd669c68-1d16-4768-b02e-befa2ff259e3)

**Comment by mkosmo** [comment]
What exactly is it doing to help with your workflow?
